platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationFileDirectory.java

```diff
@@ -151,13 +164,13 @@ private void moveConfigurationFile(Path source, Path destination) {
         return configurationFiles;
     }
 
-    private Collection<Path> listFiles() throws IOException {
+    private Collection<Path> listFiles()throws IOException {
         Collection<Path> fileNames = new ArrayList<>();
 
-        for (Path path : directoryStream) {
+        for (Path path : configDirectoryStream) {
             fileNames.add(path);
         }
-
+        configDirectoryStream.close();
```

Comment:
This method was created only to extract some code from the `init()` method and keep it smaller and was meant to be only called once. Maybe we could pass in the stream as a parameter and close and null it out in the `init()` method once we're done with it? There's not good reason to keep the list of files around after initialization so I'm not sure I want that to be a member field.